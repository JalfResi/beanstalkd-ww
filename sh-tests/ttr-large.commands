put 0 0 120 1
a
put 0 0 4294 1
a
put 0 0 4295 1
a
put 0 0 4296 1
a
put 0 0 4297 1
a
put 0 0 5000 1
a
put 0 0 21600 1
a
stats-job 1
stats-job 2
stats-job 3
stats-job 4
stats-job 5
stats-job 6
stats-job 7
quit
