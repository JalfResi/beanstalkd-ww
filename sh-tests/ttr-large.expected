INSERTED 1
INSERTED 2
INSERTED 3
INSERTED 4
INSERTED 5
INSERTED 6
INSERTED 7
OK 137
---
id: 1
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 120
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 138
---
id: 2
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 4294
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 138
---
id: 3
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 4295
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 138
---
id: 4
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 4296
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 138
---
id: 5
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 4297
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 138
---
id: 6
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 5000
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

OK 139
---
id: 7
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 21600
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

